### PR TITLE
Change output errors as JSON when type set to json

### LINKED ISSUE #141: Output errors as JSON when type is set to `json`
Currently there is an option to set the output type to JSON to get stats in JSON format.

Example: https://github-readme-streak-stats.herokuapp.com/?user=DenverCoder1&type=json

If there is an error (eg. invalid username, etc.), currently the error card in SVG gets displayed.

It may be best if instead of returning the SVG it echos the response in JSON format, for example:

```json
{
    "error": "User could not be found."
}
```

**Note:** It may be advisable to wait for #140 to be merged before starting in order to avoid merge conflicts.

### PR DESCRIPTION
**Pull request by menghif**
## Description

Fixes #141 

This PR changes the output error to JSON when there is an error and the type is set to json:
```
{
"error": "User could not be found."
}
```
### Type of change

- [x] Bug fix (added a non-breaking change which fixes an issue)
- [ ] New feature (added a non-breaking change which adds functionality)
- [ ] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [x] Tested locally with a valid username
- [x] Tested locally with an invalid username
- [x] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/index.php [hunk 1]
```diff
@@ -48,6 +48,13 @@
     $contributions = getContributionDates($contributionGraphs);
     $stats = getContributionStats($contributions);
 } catch (InvalidArgumentException $error) {
+    if ($requestedType === "json") {
+        // set content type to JSON
+        header('Content-Type: application/json');
+        // echo JSON error message
+        echo json_encode(array("error"=>"User could not be found."));
```
Comment (DenverCoder1) [inline src/index.php:55]:
```suggestion
        echo json_encode(array("error" => $error->getMessage()));
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]
There is also another error card around line 24 of index.php

**Review by DenverCoder1 — APPROVED** [review]
Works great! Thanks for contributing! :tada:

**Comment by menghif** [comment]
Hey, sorry it's my first time working with PHP but it's a great learning opportunity!
I just added the missing json error output. Let me know if there is anything missing.

**Comment by DenverCoder1** [comment]
It looks good. :+1: Usually I'd recommend putting any code that's repeated into a new function, but we've got #163 open that's taking care of that issue, so hopefully they will be able to incorporate this into their solution as well.

**Pull request merged**